tools/metadata.json

```diff
@@ -1,3 +1,3 @@
 {
-    "ReleaseTag": "v6.0.0-beta.9"
+    "ReleaseTag": "v6.1.0-preview.1"
```

Comment:
We cannot change this key/value pair because `install-powershell.ps1` depends on it to get the latest release package.
But we can add a new key/value pair for the daily build to use.

Note that the value for `ReleaseTag` should be updated to `v6.0.0-rc`, in both master and 6.0.0 branches. It was missed from the release update.